pkgs/development/python-modules/quamash/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k, pytest, pyqt5, pyqt ? pyqt5 }:
+buildPythonPackage rec {
+  pname = "Quamash";
```

Comment:
```suggestion
  pname = "quamash";
```
We also don't like uppercase letters in the `name` attribute

> https://nixos.org/nixpkgs/manual/#sec-package-naming
> Don’t use uppercase letters in the name attribute — e.g., "mplayer-1.0rc2" instead of "MPlayer-1.0rc2".